desktop/src/main/java/bisq/desktop/splash/SplashView.java

```diff
@@ -77,9 +102,68 @@ protected void onViewAttached() {
 
     @Override
     protected void onViewDetached() {
-        transportStatusLabel.textProperty().unbind();
+        this.unbindNetworkState(clearStatusHBox, clearStatusLabel, clearIcon);
+        this.unbindNetworkState(torStatusHBox, torStatusLabel, torIcon);
+        this.unbindNetworkState(i2pStatusHBox, i2pStatusLabel, i2pIcon);
         progressBar.progressProperty().unbind();
         progressBar.setProgress(0);
         appStateSubscription.unsubscribe();
     }
+
+    private HBox createNetworkStatusBox() {
+        final double height = 24;
```

Comment:
According to our code conventions we do not use `final` in local variables.

Reply:
Do you refer to any docs for code conventions?